Adding QML tutorial (#702)
MicrosoftDocs/quantum-docs-pr: articles/libraries/machine-learning/index.md

# Introduction to the Quantum Machine Learning Library

The Quantum Machine Learning Library is a high-level API, written in Q#, that provides the first playground to run hybrid quantum/classical machine learning experiments. From this library you can expect:

> - Basic tools to load your own data to classify with quantum simulators

Reviewer: I would list the features of the library here, and put the limitations in a paragraph below

> - Slow performance compared to current classical machine learning frameworks (remember that this library is highly experimental).

Reviewer 2: I think this is correct, but it's not because the library is experimental so much as that the task of simulating a quantum device running a QML algorithm is inherently harder than what classical ML libraries do. Perhaps we could rephrase this in a more positive way by expressing that this is intended to help prepare for quantum hardware by enabling simulation now?

> - Samples and tutorials to get introduced to the field of Quantum Machine Learning.

Reviewer 2: ```suggestion
- Samples and tutorials to get introduced to the field of quantum machine learning.
```

> The purpose of this documentation is to provide a concise introduction to machine learning tools (written in Q\#) for hybrid quantum/classical learning. We begin with a brief introduction to machine learning concepts and specifically their realization in quantum circuit centric classifiers (also known as quantum sequential classifiers). Then we introduce with a set of tutorials the basics to start using the tools provided by the library. We then discuss the training and validation methods for such classifiers and how they map on specific Q\# operations provided by the library.

Reviewer 2: ```suggestion
The purpose of this documentation is to provide a concise introduction to machine learning tools (written in Q\#) for hybrid quantum/classical learning. We begin with a brief introduction to machine learning concepts and specifically their realization in quantum circuit centric classifiers (also known as quantum sequential classifiers). We then introduce with a set of tutorials the basics to start using the tools provided by the library. We then discuss the training and validation methods for such classifiers and how they map on specific Q\# operations provided by the library.
```

> The purpose of this documentation is to provide a concise introduction to machine learning tools (written in Q\#) for hybrid quantum/classical learning. We begin with a brief introduction to machine learning concepts and specifically their realization in quantum circuit centric classifiers (also known as quantum sequential classifiers). We then introduce a set of tutorials on the basics to start using the tools provided by the library. We also discuss the training and validation methods for such classifiers and how they translate into specific Q\# operations provided by the library.

Reviewer: ```suggestion
The purpose of this documentation is to provide an introduction to machine learning tools (written in Q\#) for hybrid quantum/classical learning, including:
- An introduction to machine learning concepts and how to apply them with quantum programming
- Tutorials to help you learn xxx (enumerate the topics ...)
- A catalog of training and validation methods for the classifiers and how they translate into specific Q\# operations
```
